paramiko_jump/client.py

```diff
@@ -102,35 +103,34 @@ def _auth(
         )
 
     def connect(
-            self,
-            hostname,
-            port=SSH_PORT,
-            username=None,
-            password=None,
-            pkey=None,
-            key_filename=None,
-            timeout=None,
-            allow_agent=True,
-            look_for_keys=True,
-            compress=False,
-            sock=None,
-            gss_auth=False,
-            gss_kex=False,
-            gss_deleg_creds=True,
-            gss_host=None,
-            banner_timeout=None,
-            auth_timeout=None,
-            gss_trust_dns=True,
-            passphrase=None,
-            disabled_algorithms=None,
+        self,
+        hostname,
+        port=SSH_PORT,
+        username=None,
+        password=None,
+        pkey=None,
+        key_filename=None,
+        timeout=None,
+        allow_agent=True,
+        look_for_keys=True,
+        compress=False,
+        sock=None,
+        gss_auth=False,
+        gss_kex=False,
+        gss_deleg_creds=True,
+        gss_host=None,
+        banner_timeout=None,
+        auth_timeout=None,
+        gss_trust_dns=True,
+        passphrase=None,
+        disabled_algorithms=None,
     ):  # pylint: disable=R0913,R0914
         if self._jump_session is not None:
             if sock is not None:
-                raise ValueError('jump_session= and sock= are mutually '
-                                 'exclusive')
+                raise ValueError("jump_session= and sock= are mutually " "exclusive")
```

Comment:
I really prefer single quotes, but most of all I prefer to not have alternating styles in a single project.